geomesa-kafka/geomesa-kafka-datastore/pom.xml

```diff
@@ -98,4 +98,19 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
```

Comment:
you can still build with -DskipTest, just not with -Dmaven.test.skip. We already have this problem with geomesa-accumulo-datastore i believe... In practice, once you build once you can use the more aggressive test skip.